odl/discr/lp_discr.py

```diff
@@ -536,27 +624,38 @@ def uniform_discr_fromspace(fspace, nsamples, exponent=2.0, interp='nearest',
         The parameter :math:`p` in :math:`L^p`. If the exponent is not
         equal to the default 2.0, the space has no inner product.
     interp : `str`, optional
-            Interpolation type to be used for discretization.
+        Interpolation type to be used for discretization.
 
             'nearest' : use nearest-neighbor interpolation (default)
 
             'linear' : use linear interpolation (not implemented)
     impl : {'numpy', 'cuda'}
         Implementation of the data storage arrays
+
+    nodes_on_bdry : `bool` or boolean array-like
+        If `True`, place the outermost grid points at the boundary. For
+        `False`, they are shifted by half a cell size to the 'inner'.
+        If an array-like is given, it must have shape ``(ndim, 2)``,
+        where ``ndim`` is the number of dimensions. It defines per axis
+        whether the leftmost (first column) and rightmost (second column)
+        nodes node lie on the boundary.
+        Default: `False`
     order : {'C', 'F'}  (Default: 'C')
         Axis ordering in the data storage
     dtype : dtype
         Data type for the discretized space
 
-        Default for 'numpy': 'float64' / 'complex128'
-        Default for 'cuda': 'float32' / TODO
+            Default for 'numpy': 'float64' / 'complex128'
+
+            Default for 'cuda': 'float32' / (not implemented)
+
     weighting : {'simple', 'consistent'}
         Weighting of the discretized space functions.
 
-        'simple': weight is a constant (cell volume)
+            'simple': weight is a constant (cell volume)
 
-        'consistent': weight is a matrix depending on the
-        interpolation type
+            'consistent': weight is a matrix depending on the
+            interpolation type (not implemented)
```

Comment:
That's a good question. I'm getting the feeling this will never be implemented due to lacking need. Remove altogether?